precompile/living-assets/src/tests.rs

```diff
@@ -60,14 +60,14 @@ fn create_collection_should_return_address() {
 	assert!(result.is_ok());
 	// check that the output is the collection id 0
```

Comment:
Outdates comment still using "collection id". Perhaps rephrase: "	// check that the output is the collection address 5, with a 1-bit at pos 160"

Reply:
sgtm